mojave/structures/ores.dm

```diff
@@ -0,0 +1,266 @@
+	name = "pitchblende nuggets"
+	desc = "A hard lump of uranium, Useless now."
+	singular_name = "pitchblende nugget"
+	icon = 'mojave/icons/objects/crafting/materials_inventory.dmi'
+	icon_state = "uranium"
+	merge_type = /obj/item/stack/sheet/ms13/nugget_uranium
+	amount = 1
+	max_amount = 16
+
+/obj/item/stack/sheet/ms13/nugget_uranium/two
+	amount = 2
+
+/obj/item/stack/sheet/ms13/nugget_coal
+	name = "coal chunks"
+	desc = "A hard lump of coal, careful with any flames."
+	singular_name = "coal chunk"
+	icon = 'mojave/icons/objects/crafting/materials_inventory.dmi'
+	icon_state = "coal"
+	merge_type = /obj/item/stack/sheet/ms13/nugget_coal
+	amount = 1
+	max_amount = 16
+
+/obj/item/stack/sheet/ms13/nugget_coal/two
+	amount = 2
+
+/obj/item/stack/sheet/ms13/nugget_copper
+	name = "copper ore nuggets"
+	desc = "Full of potential copper. Useless now."
+	singular_name = "copper ore nugget"
+	icon = 'mojave/icons/objects/crafting/materials_inventory.dmi'
+	icon_state = "copper"
+	merge_type = /obj/item/stack/sheet/ms13/nugget_copper
+	amount = 1
+	max_amount = 16
+
+/obj/item/stack/sheet/ms13/nugget_copper/two
+	amount = 2
+
+/obj/item/stack/sheet/ms13/nugget_lead
+	name = "lead ore nuggets"
+	desc = "Full of potential lead. Useless now."
+	singular_name = "lead ore nugget"
+	icon = 'mojave/icons/objects/crafting/materials_inventory.dmi'
+	icon_state = "lead"
+	merge_type = /obj/item/stack/sheet/ms13/nugget_lead
+	amount = 1
+	max_amount = 16
+
+/obj/item/stack/sheet/ms13/nugget_lead/two
+	amount = 2
+
+/obj/item/stack/sheet/ms13/nugget_alu
+	name = "aluminum ore nuggets"
+	desc = "Full of potential aluminum. Useless now."
+	singular_name = "aluminum ore nugget"
+	icon = 'mojave/icons/objects/crafting/materials_inventory.dmi'
+	icon_state = "aluminum"
+	merge_type = /obj/item/stack/sheet/ms13/nugget_alu
+	amount = 1
+	max_amount = 16
+
+/obj/item/stack/sheet/ms13/nugget_alu/two
+	amount = 2
+
+/obj/item/stack/sheet/ms13/nugget_silver
+	name = "silver ore nuggets"
+	desc = "Full of potential silver. Useless now."
+	singular_name = "silver ore nugget"
+	icon = 'mojave/icons/objects/crafting/materials_inventory.dmi'
+	icon_state = "silver"
+	merge_type = /obj/item/stack/sheet/ms13/nugget_silver
+	amount = 1
+	max_amount = 16
+
+/obj/item/stack/sheet/ms13/nugget_silver/two
+	amount = 2
+
+/obj/item/stack/sheet/ms13/nugget_gold
+	name = "gold ore nuggets"
+	desc = "Full of potential gold. Useless now."
+	singular_name = "gold ore nugget"
+	icon = 'mojave/icons/objects/crafting/materials_inventory.dmi'
+	icon_state = "gold"
+	merge_type = /obj/item/stack/sheet/ms13/nugget_gold
+	amount = 1
+	max_amount = 16
+
+/obj/item/stack/sheet/ms13/nugget_gold/two
+	amount = 2
+
+//Deposit generation
+
+#define DEPOSIT_SPAWN_LIST list(/obj/structure/ms13/ore_deposit/gold = 1, /obj/structure/ms13/ore_deposit/silver = 2, /obj/structure/ms13/ore_deposit/alu = 6, /obj/structure/ms13/ore_deposit/lead = 6, /obj/structure/ms13/ore_deposit/copper = 5)
```

Comment:
You should add coal to this provided it's functioning properly and is all there. I assume maybe you originally left it out because you weren't able to find a use for it but I'll probably be able to use it for some things.